x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/action/user/PutUserRequestBuilderTests.java

```diff
@@ -221,8 +239,7 @@ public void testWithPasswordHashThatsNotReallyAHash() throws IOException {
                     .request();
             }
         );
-        assertThat(ex.getMessage(), containsString(Hasher.NOOP.name()));
-        assertThat(ex.getMessage(), containsString(systemHasher.name()));
+        assertThat(ex.getMessage(), containsString("Provided password hash is either using unsupported format or it is a clear text"));
```

Comment:
nit: can we also test the inverse ... configured for plain text and an input with plain text -> no error.

Reply:
Good point. I will add the test.